lambda_handlers/handlers/event_handler.py

```diff
@@ -0,0 +1,39 @@
+from lambda_handlers import formatters
+from lambda_handlers.validators.validator import Validator
+from lambda_handlers.handlers.lambda_handler import LambdaHandler
+from lambda_handlers.handlers.mixins.formatting_mixin import FormattingMixin
+from lambda_handlers.handlers.mixins.validation_mixin import ValidationMixin
+
+
+class EventHandler(LambdaHandler, ValidationMixin, FormattingMixin):
+    """
+    Decorator class to facilitate the definition of AWS Lambda handlers with:
+        - input validation,
+        - output formatting.
+
+    Parameters
+    ----------
+    input_format: Callable
+        Formatter callable to parse the input event.
```

Comment:
Could you please mention here the defaults?

Reply:
done